### DISCUSSION TITLE
NoSQL vs NoSQL Databases

### DISCUSSION THREAD

**Discussion by RayMathew** [Buzzpy/Dev-Encyclopedia#83] [Q&A] [answered]
I'm not sure there needs to be 2 separate cards for NoSQL as well as NoSQL Databases. @Buzzpy what do you say?

**Comment by Buzzpy** [comment, marked as answer]
I agree. Let's keep the NoSQL card.

**Reply by RayMathew** [reply]
This is complete https://github.com/Buzzpy/Dev-Encyclopedia/pull/99
